### PR TITLE
fix  nighty build of branch3.1

### PR DESCRIPTION
**Pull request by brianhu2006**
1. cherry pick the two commits
	e9c5cb6
	8538d69
2. update kvm version

### CODE REVIEW

File: roles/nova-common/defaults/main.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ nova:
   scheduler_host_subset_size: 1
   libvirt_bin_version: 1.3.1-1ubuntu10.8~cloud0
   python_libvirt_version: 1.3.1-1ubuntu1~cloud0
-  qemu_kvm_version: 2.0.0+dfsg-2ubuntu1.30
+  qemu_kvm_version: 1:2.5+dfsg-5ubuntu10.9~cloud0
```
Comment (nirajdp76) [inline roles/nova-common/defaults/main.yml]: why do we need to cherry pick qemu kvm version bump?

Comment (nirajdp76) [inline roles/nova-common/defaults/main.yml]: seems like qemu-kvm version in ubuntu mirror got bumped to 2.0.0+dfsg-2ubuntu1.33. Your changes looks fine. Waiting on QA team regression results before merging this.

**Comment by epwn** [comment]
please squash your commits

**Comment by nirajdp76** [comment]
qa passed regression bucket for nova and cinder with qemu-kvm version bump.

**Comment by brianhu2006** [comment]
because the two commits are cherry pick and new commit is to update kvm version, those commits are not suqash.

**Pull request merged**